ArkGameExample/SceneDelegate.swift

```diff
@@ -55,20 +56,35 @@ extension SceneDelegate {
     func defineArkBlueprint() -> ArkBlueprint {
         // Define game with blueprint here.
         let arkBlueprint = ArkBlueprint()
-            .ecsSetup({ ecsContext in
+            .stateSetup({ ecsContext, eventContext in
                 _ = ecsContext.createEntity(with: [
```

Comment:
We can consider adding `@discardableresult` to the `.createEntity` method to omit the _ =